src/modules/login/RegisterForm.tsx

```diff
@@ -103,16 +104,20 @@ const RegisterForm = ({ done }: IProps) => {
       email: values.email,
       phone: values.phone,
       inCell: values.inCell,
-      joinCell: values.joinCell
+      joinCell: values.joinCell,
     };
 
     post(
       remoteRoutes.register,
       toSave,
-      data => {
-        Toast.info("Operation successful");
-        actions.resetForm();
-        if (done) done();
+      (data) => {
+        if (isEmpty(data)) {
+          Toast.error("This Email address is already in use");
+        } else {
+          Toast.info("Operation successful");
+          actions.resetForm();
+          if (done) done();
+        }
       },
       undefined,
```

Comment:
The error should be handled here... 
```
(error)=>{
Toast.info(error.message);
}
```

Reply:
Thanks Timo. The code has been amended to handle this appropriately.